scripts/create_dataset.py

```diff
@@ -0,0 +1,108 @@
+import os
```

Comment:
Can you add a `#!` line to this file, to make it clear if it should be run with Python 2 or Python 3?

One of:
1. `#!/usr/bin/env python2`
2. `#!/usr/bin/env python3`

Then also `chmod a+x` the file, so it can be executed directly.